src/test/regress/expected/pg16.out

```diff
@@ -127,6 +127,138 @@ HINT:  Consider specifying a name for the statistics
+COPY copy_default FROM stdin WITH (format csv, quote '"', default 'test"test');
+ERROR:  CSV quote character must not appear in the DEFAULT specification
+-- NULL and DEFAULT spec must be different
+COPY copy_default FROM stdin WITH (default '\N');
+ERROR:  NULL specification and DEFAULT specification cannot be the same
+-- cannot use DEFAULT marker in column that has no DEFAULT value
+COPY copy_default FROM stdin WITH (default '\D');
+ERROR:  unexpected default marker in COPY data
+DETAIL:  Column "id" has no default value.
+CONTEXT:  COPY copy_default, line 1: "\D	value	'2022-07-04'"
+COPY copy_default FROM stdin WITH (format csv, default '\D');
+ERROR:  unexpected default marker in COPY data
+DETAIL:  Column "id" has no default value.
+CONTEXT:  COPY copy_default, line 1: "\D,value,2022-07-04"
+-- The DEFAULT marker must be unquoted and unescaped or it's not recognized
+COPY copy_default FROM stdin WITH (default '\D');
+SELECT * FROM copy_default ORDER BY id;
+ id | text_value |         ts_value
+---------------------------------------------------------------------
+  1 | test       | Mon Jul 04 00:00:00 2022
+  2 | \D         | Mon Jul 04 00:00:00 2022
+  3 | "D"        | Mon Jul 04 00:00:00 2022
+(3 rows)
+
+TRUNCATE copy_default;
+COPY copy_default FROM stdin WITH (format csv, default '\D');
+SELECT * FROM copy_default ORDER BY id;
+ id | text_value |         ts_value
+---------------------------------------------------------------------
+  1 | test       | Mon Jul 04 00:00:00 2022
+  2 | \\D        | Mon Jul 04 00:00:00 2022
+  3 | \D         | Mon Jul 04 00:00:00 2022
+(3 rows)
+
+TRUNCATE copy_default;
+-- successful usage of DEFAULT option in COPY
+COPY copy_default FROM stdin WITH (default '\D');
+SELECT * FROM copy_default ORDER BY id;
+ id | text_value |         ts_value
+---------------------------------------------------------------------
+  1 | value      | Mon Jul 04 00:00:00 2022
+  2 | test       | Sun Jul 03 00:00:00 2022
+  3 | test       | Tue Jul 05 00:00:00 2022
+(3 rows)
+
+TRUNCATE copy_default;
+COPY copy_default FROM stdin WITH (format csv, default '\D');
+SELECT * FROM copy_default ORDER BY id;
+ id | text_value |         ts_value
+---------------------------------------------------------------------
+  1 | value      | Mon Jul 04 00:00:00 2022
+  2 | test       | Sun Jul 03 00:00:00 2022
+  3 | test       | Tue Jul 05 00:00:00 2022
+(3 rows)
+
+TRUNCATE copy_default;
+\c - - - :worker_1_port
+COPY pg16.copy_default FROM stdin WITH (format csv, default '\D');
+SELECT * FROM pg16.copy_default ORDER BY id;
+ id | text_value |         ts_value
+---------------------------------------------------------------------
+  1 | value      | Mon Jul 04 00:00:00 2022
+  2 | test       | Sun Jul 03 00:00:00 2022
+  3 | test       | Tue Jul 05 00:00:00 2022
+(3 rows)
+
+\c - - - :master_port
+TRUNCATE pg16.copy_default;
+\c - - - :worker_2_port
+COPY pg16.copy_default FROM stdin WITH (format csv, default '\D');
```

Comment:
as we sure that there are local shards here? if not, please consider adjusting the "id".

I'd be even fine to set citus.log_local_commands TO ON;

Reply:
We are sure one of the two workers has shards, so I just connected to both of them.